src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -48,6 +49,7 @@ export class AddCluster extends React.Component {
   componentDidMount() {
     clusterStore.setActive(null);
     this.setKubeConfig(userStore.kubeConfigPath);
+    appEventBus.emit({ name: "cluster-add", action: "start" });
```

Comment:
do we already have some syntax recommendation `cluster-add` vs `clusterAdd`?

Reply:
Not sure about recommendations but I see that we already has some kebab-case notations, for example in `src/main/node-shell-session.ts`